examples/v1beta1/pipelineruns/pipelinerun-with-uses-individual-steps.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generatestep: my-release-
+spec:
+  serviceAccountName: 'default'
+  pipelineSpec:
+    tasks:
+    - taskSpec:
```

Comment:
Any reason for not using a `TaskRun` here instead ? A `PipelineRun` with only one task makes little sense 😅 .

Reply:
no particular reason ;) we've always tended to use PipelineRun's so it was an example I had to hand; but folks should be able to use TaskRun too. I'll add the same examples as TaskRuns too...

Reply:
added examples for Task/TaskRun too